sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala

```diff
@@ -300,6 +302,48 @@ class StreamSuite extends StreamTest {
       q.stop()
     }
   }
+
+  test("StreamingQuery should be Serializable but cannot be used in executors") {
```

Comment:
Why is this in StreamSuite. This is better to be in StreamingQuerySuite as its related to the StreamingQuery.